grow-website-gatsby/src/pages/components/Workshop/Workshop.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { Fragment } from "react"
+import Moment from "moment"
+import AnchorLink from "react-anchor-link-smooth-scroll"
+import { Event } from "schema-dts"
+import { JsonLd } from "react-schemaorg"
+
+import * as styles from "../../Index.module.scss"
+import WorkshopData from "./WorkshopData"
+
+const Workshop = () => {
+  return (
+    <div id="workshop">
+      <div className={styles.sectionWorkshop}>
+        <div className={styles.headingLarge}>Workshops</div>
+        <div className={styles.headingXSmall}>Upcoming Event</div>
+        <div className={styles.workshopImageContainer}>
+          <div className={styles.ratioContent}>
+            <img
+              className={styles.workshopImage}
+              src={WorkshopData.image}
+              alt={WorkshopData.imageAltText}
+            />
+          </div>
+        </div>
+        <div className={styles.headingMedium}>{WorkshopData.name}</div>
+        <div className={styles.uWidthSixtyPercent}>
+          <div className={styles.headingSubtitle}>
+            {Moment(WorkshopData.dateTime).format("ha, dddd do MMMM YYYY")} |
+            {WorkshopData.locationName}, {WorkshopData.locationAddress}
+          </div>
+          {WorkshopData.description.map((paragraph, i) => {
+            return (
+              <p className={styles.paragraphSmall} key={i}>
+                {paragraph}
+              </p>
+            )
+          })}
+          <div className={styles.uMarginBottomS}>
+            <AnchorLink
+              className={WorkshopData.isWorkshopOpen ? "" : styles.linkDisabled}
+            >
+              Apply for this workshop
+            </AnchorLink>
+          </div>
+          {WorkshopData.isWorkshopOpen ? (
```

Comment:
@catepalmer should the application link be inside of the conditional? Otherwise, I'd say it's probably cleaner to write this as 

```
{ !WorkshopData.isWorkshopOpen && (
  .....
) }
```